src/middlewared/middlewared/etc_files/nfsd.py

```diff
@@ -0,0 +1,139 @@
+from collections import defaultdict
+import ipaddress
+import logging
+import os
+import re
+
+from middlewared.utils import run
+
+logger = logging.getLogger(__name__)
+
+IP_REGEX = "^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])$"
```

Comment:
We should probably be restricting to IPv4 in the plugin.

Reply:
This is IPv4, isn't it?

Reply:
But we are accepting IPV6 in the plugin, aren't we?

Reply:
@william-gr fixed